### ISSUE TITLE
Mini player visual modes

### ISSUE THREAD

**Issue by chrismou** [radiant-player/radiant-player-electron#9]
Radiant V1 has the ability to click the album artwork in the mini player to switch between a a minimal & artwork-centric mode (this may be controllable via the settings, but I'm not sure that's necessarily required.

**Comment by matthewdias** [comment]
IMO it doesn't make alot of sense to have an artwork-centric mode for something that's not permanently visible.

**Comment by chrismou** [comment]
More of a nice to have tbh. Not necessarily required, but it looks quite cool and would be putting existing functionality from the old app

**Comment by jacobwgillespie** [comment]
Yeah, I don't think it'd be too hard to implement actually - there's some technical details with window sizing that would have to be worked around (the window would actually need to be max-size when created, and then the lower region made transparent when it was in the compact mode) but it shouldn't be too difficult.
